lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -95,11 +95,11 @@ def trigger
         puppet_trigger['day_of_week'] = PuppetX::PuppetLabs::ScheduledTask::Trigger::V1::Day.bitfield_to_names(trigger['type']['days_of_week'])
       when Win32::TaskScheduler::TASK_TIME_TRIGGER_MONTHLYDATE
         puppet_trigger['schedule'] = 'monthly'
-        puppet_trigger['months']   = PuppetX::PuppetLabs::ScheduledTask::Trigger::V1::Month.months_from_bitfield(trigger['type']['months'])
+        puppet_trigger['months']   = PuppetX::PuppetLabs::ScheduledTask::Trigger::V1::Month.bitfield_to_indexes(trigger['type']['months'])
```

Comment:
Not sure how I feel about `indexes`.  Index into what?

Reply:
They are month indexes .. i.e. 1 through 12. I couldn't readily think of a better name, but open to suggestions if you have any.

Reply:
https://en.wikipedia.org/wiki/Month#Julian_and_Gregorian_calendars doesn't have too much else...

Is `bitmask_to_numbers` any better than `bitmask_to_indexes`?

Reply:
Longer, but possibly `bitmask_to_month_index`?